lib/ansible/modules/packaging/os/portage.py

```diff
@@ -326,13 +326,20 @@ def emerge_packages(module, packages):
         module.fail_json(msg='Use only one of usepkg, usepkgonly')
 
     emerge_flags = {
-        'jobs': '--jobs=',
-        'loadavg': '--load-average ',
+        'jobs': '--jobs',
+        'loadavg': '--load-average',
     }
 
     for flag, arg in emerge_flags.items():
-        if p[flag] is not None:
-            args.append(arg + str(p[flag]))
+        flag_val = p[flag]
+        if flag_val is None or flag_val is False:
+            continue
+
+        if flag_val is True:
+            args.append(arg)
+            continue
+
+        args.extend((arg, str(flag_val)))
```

Comment:
use to_native (module_utils._text) instead of str, it deals with py2/py3 compatiblity

Reply:
@bcoca done, thanks!